src/devices/bus/coco/coco_ssc.cpp

```diff
@@ -550,17 +508,19 @@ void cocossc_sac_device::sound_stream_update(sound_stream &stream, std::vector<r
 
 	double n = dst.samples();
 
-	for (int sampindex = 0; sampindex < n; sampindex++)
-	{
-		m_rms[m_index] += src.get(sampindex) * src.get(sampindex);
-		dst.put(sampindex, src.get(sampindex));
-	}
+	if( n > 0 ) {
```

Comment:
Please use the same style consistently.  This `if` doesn’t even match the `for` below it.